### ISSUE TITLE
Class not found: my_project.NodeTree.001

### ISSUE THREAD

**Issue by Devination** [luboslenco/zblend#9]
If you make more than 1 NodeTree in a zblend .blend file then you will get this error on build:

"Haxe stderr: ..\Sources/Main.hx:10: characters 8-47 : Class not found : my_project.NodeTree.001"
"Haxe process exited with code 1"

Even if you do not use NodeTree.001 anywhere, it will still give that error. Also, renaming NodeTree.001 to something else like "NodeTree2" will still give the error message on build using the old name "NodeTree.001"
![capture](https://cloud.githubusercontent.com/assets/8147765/7217842/7fb96c0a-e5fc-11e4-8c72-53562a8d0454.PNG)

**Comment by luboslenco** [comment]
Good catch! Should be fixed now, the '.' was causing trouble in generated Haxe code.

'Clean' button now also removes compiled nodes, so if something goes wrong during node parsing cleaning may help get it back to working state.

**Comment by Devination** [comment]
Old project was still getting that error even after a clean so I started a new one with same node setup. No longer errors on build but if I make something in NodeTree.001 and assign that one to the object in the traits view then the cube doesn't render and I get these errors:

Firefox: "TypeError: cl is not a constructor kha.js:351:0"

Chrome: "kha.js:351 Uncaught TypeError: object is not a function"
"http://127.0.0.1:8000/favicon.ico Failed to load resource: the server responded with a status of 404 (File not found)"

You can go back to the first nodeTrue, make edits and use that one and it works fine though

**Comment by luboslenco** [comment]
Silly me, forgot to also take care of '.' character in nodes trait. Now it should really work(hopefully).

Let me know if you get any ideas on what to do next with nodes. Thinking about adding some kind of conditions. :)

**Comment by Devination** [comment]
It works thanks! I tried adding an 'Input' node to get mouse input but to no avail :( I started by extending the FloatNode like you did for SineNode and it worked with 1 value. But once I changed the variable name from 'f' to 'x' it stopped working. Adding in another variable 'y' also didn't work.

I've got my code up here if you have time to review it: https://drive.google.com/open?id=0Bwiz0Tss2NSqfkZESWhEQmpEcDFreGlSRk5fM3lYdVVJazdrWWRiQVphb1pNWmw5aUhVUjg&authuser=0

I'm a little new to Haxe but I'd love to contribute to ZBlend once I get it figured out :p
